sr_example/scripts/sr_example/sr_left_examples_unsafe.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
```

Comment:
Needs an associated markdown explanation in https://github.com/shadow-robot/sr-ros-interface/tree/indigo-devel/sr_example/doc